### PR TITLE
feat(sast_warnings): add invalidSemVer translations

### PR DESCRIPTION
**Pull request by fabnguess**
As part of PR 139 [https://github.com/NodeSecure/scanner/pull/139](url) it was necessary to create an invalidSemVer token for the French and English languages

### CODE REVIEW

File: languages/english.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ const ui = {
 
 const warnings = {
   disable_scarf: "This dependency could collect data against your will so think to disable it with the env var: SCARF_ANALYTICS",
-  keylogging: "This dependency can retrieve your keyboard and mouse inputs. It can be used for 'keylogging' attacks/malwares."
+  keylogging: "This dependency can retrieve your keyboard and mouse inputs. It can be used for 'keylogging' attacks/malwares.",
```
Comment (PierreDemailly) [inline languages/english.js:120]: This change should not be needed, right ?

**Review by PierreDemailly — CHANGES_REQUESTED** [review]
@fabnguess Please run `npm run build:documentation` to update docs

File: languages/english.js (hunk 1 above)
Comment (fabnguess) [reply languages/english.js]: Is done

**Comment by PierreDemailly** [comment]
@fabnguess Can you check for the broken test ? It's not related to your changes but it should not be hard to be fixed

**Comment by fabnguess** [comment]
ok i'm looking at this

**Comment by PierreDemailly** [comment]
@fabnguess You can rebase #97 once it's merged ;) I wanted to udpate deps anyway so I fixed it

**Comment by fabnguess** [comment]
Hi @PierreDemailly , well noted, I will keep an eye disappointed

**Review by PierreDemailly — APPROVED** [review]

**Review by tony-go — APPROVED** [review]

**Review by fraxken — APPROVED** [review]

**Pull request merged**